testnet/stacks-node/src/neon_node.rs

```diff
@@ -966,6 +956,12 @@ fn spawn_miner_relayer(
     Ok(())
 }
 
+enum LeaderKeyRegistrationState {
+    None,
```

Comment:
Style nit -- can we avoid using reserved Rust keywords?  Maybe `Nothing` instead of `None`?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2318/commits/040777078f052f63d1654e8d4beb6fc11b878237